src/appMain/main.cc

```diff
@@ -103,6 +103,10 @@ int32_t main(int32_t argc, char** argv) {
   }
 
   PLATFORM_INIT(argc, argv);
+#ifdef QT_PORT
+  QThreadPool * pool = QThreadPool::globalInstance();
+  pool->setMaxThreadCount(MAX_THREADS);
+#endif
```

Comment:
I recommend to move it in ``utils::threads``
It could be external static initialization method *or* internal static initialization in ``threads_qt.cc``:
```
static bool is_thread_init = init();
init() {
poll = ...
};
```